docs/Preparing_your_workstation.adoc

```diff
@@ -108,6 +108,8 @@ brew install awscli
 
 ==== Configure the EC2 Credentials
 
+In order to be able to deploy AgnosticD configs on AWS, you will need an IAM user into your AWS account. If you are new to AWS, you can find the instructions link:here[https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html].
```

Comment:
Oops, I think actually you have this backwards. I think it should be

`link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html[Appropriate Text^]`